### ISSUE TITLE
Results are not same as per paper.

### ISSUE THREAD

**Issue by redhat12345** [thuml/transfer-caffe#9]
I have run your code (AlexNet) got 68.67% accuracy (target data) (adversarial). ANd always get fc7_mmd_loss = 0 and label_mmd_loss = 0. Could you help me out to solve this problem?

I1005 07:16:44.324632 17598 solver.cpp:407]     Test net output #0: accuracy = 0.686792
I1005 07:16:44.494951 17598 solver.cpp:231] Iteration 2700, loss = 0.00387742
I1005 07:16:44.494974 17598 solver.cpp:247]     Train net output #0: fc7_mmd_loss = 0 (* 0.3 = 0 loss)
I1005 07:16:44.494979 17598 solver.cpp:247]     Train net output #1: label_mmd_loss = 0 (* 0.3 = 0 loss)
I1005 07:16:44.494998 17598 solver.cpp:247]     Train net output #2: softmax_loss = 0.00387735 (* 1 = 0.00387735 loss)
I1005 07:16:44.495003 17598 sgd_solver.cpp:106] Iteration 2700, lr = 0.00112453

**Comment by caozhangjie** [comment]
JAN code are transferred to the Xlearn repository here https://github.com/thuml/Xlearn
